prac_06/car.py

```diff
@@ -0,0 +1,36 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, name="", fuel=0):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
+        self.name = name
+        self.fuel = fuel
+        self._odometer = 0
+
+    def __str__(self):
+        """Return Car as string"""
```

Comment:
It might be a good idea to put a full stop at the end.

Reply:
Added 👍